README.md

```diff
@@ -7,7 +7,19 @@
 # mermaid [![Build Status](https://travis-ci.org/mermaid-js/mermaid.svg?branch=master)](https://travis-ci.org/mermaid-js/mermaid) [![NPM](https://img.shields.io/npm/v/mermaid)](https://www.npmjs.com/package/mermaid) [![Coverage Status](https://coveralls.io/repos/github/mermaid-js/mermaid/badge.svg?branch=master)](https://coveralls.io/github/mermaid-js/mermaid?branch=master) [![Join our Slack!](https://img.shields.io/static/v1?message=join%20chat&color=9cf&logo=slack&label=slack)](https://join.slack.com/t/mermaid-talk/shared_invite/enQtNzc4NDIyNzk4OTAyLWVhYjQxOTI2OTg4YmE1ZmJkY2Y4MTU3ODliYmIwOTY3NDJlYjA0YjIyZTdkMDMyZTUwOGI0NjEzYmEwODcwOTE) [![This project is using Percy.io for visual regression testing.](https://percy.io/static/images/percy-badge.svg)](https://percy.io/Mermaid/mermaid)
 <!-- <Main description> -->
 __Generate diagrams, charts, graphs or flows from markdown-like text via javascript.__
-See our [documentation](http://mermaid-js.github.io/mermaid/) and start simplifying yours. Play in our [live editor](https://mermaidjs.github.io/mermaid-live-editor/) or jump straight to the [installation and usage](http://mermaid-js.github.io/mermaid/#/usage).
+See our [documentation](http://mermaid-js.github.io/mermaid/) and start simplifying yours. Play in our [live editor](https://mermaidjs.github.io/mermaid-live-editor/) or jump straight to the installation:
+
+```
+yarn add mermaid
+```
+
+```
+npm install mermaid
```

Comment:
Giving the commands for installation seems unnecessary (at least in the readme) since _most_ users will know them. If you feel like at a glance from the readme the users will want to know in which package managers mermaid is available (which personally I don't), we could say something like "available through`yarn` or `npm`".

Reply:
When I search for a package I'm always glad if they have a simple command line instruction in their readme, not because I don't know that there are different package managers but because I can just copy paste it and don't have to search for the right package name (which not always matches).

Reply:
Maybe "available through yarn or npm as `mermaid`"? Personally, I'm trying to aim for information density so I dislike the code blocks.

Reply:
But information density means text blocks which are not really good to read if you just fly over the content (as you usually do, I do at least).